### PR TITLE
WIP: Coveralls fix

### PR DESCRIPTION
**Pull request by gregstarr**
trying to fix coveralls in a new branch

### CODE REVIEW

**Comment by aburrell** [comment]
Well.  I had a branch going for this, but it looks like you got it merged before me.  Cheers! 🥇

File: tests/test_cmd.py [hunk 1]
```diff
@@ -6,40 +6,20 @@
...
-
-def setup_function(function):
-    try:
-        os.remove(outfile)
-    except OSError:
-        pass
-
-
-teardown_function = setup_function
-
```
Comment (aburrell) [inline tests/test_cmd.py:22]: This is needed so we don't leave files hanging around.  Did you find another way to do this that I didn't notice?

**Review by aburrell — CHANGES_REQUESTED** [review]
Some additional comments:
- The changes to .travis.yml should be made to ci/templates/.travis.yml and the run `python ci/bootstrap.py` to change the files that are templated.  Same goes for `appveyor.yml` and `tox.ini`.  Which then brings up the question, why was the tox template renamed?

**Comment by gregstarr** [comment]
Hi thanks for taking a look. I just created this PR to get CI to run, I don't think it is ready for merge yet. When I was looking at the old CI runs, I saw that the command to run pytest was this: `py37 run-test: commands[1] | py.test -vv --ignore=src '--doctest-glob=*.rst'`, which doesn't include anything to run coveralls. To simplify things I tried to get it to work without using tox. I put the command `coverage run --source=src/apexpy -m pytest -v tests/` directly in the travis yaml which seems to get coveralls to run. The problem I then encountered was that the first test_cmd.py test would fail. I tried a few things to get it to work but gave up. 

I don't have a lot of experience with the various CI tools. Why is tox being used here? It looks like there are many tox environments in the tox.ini but only the environments in the travis yaml are being run. Additionally, why are both coveralls and codecov being used?

File: tests/test_cmd.py (hunk 1 above)
Comment (gregstarr) [reply tests/test_cmd.py:22]: probably do need, see comment below

**Comment by aburrell** [comment]
Yes, it's much simpler to do coveralls without tox.  Tox has advantages, including specifying a lot of tests that you can pick and choose from to run on different systems, but is tricky to integrate with coveralls.

There are a lot of tests used here that are vaporware or not really needed.  I've gotten rid of those we don't need in the 'coveralls_update' branch.  I'll make it a draft PR so you can see what I've been doing there.  I got coveralls to work with tox in AACGMV2, but for the life of me can't remember what the trick was 😛  Research notebook fail.

**Comment by gregstarr** [comment]
Looks like you have it working in #51 I will close this

**Pull request closed without merging**